test/protocol/common.test.js

```diff
@@ -1,8 +1,11 @@
+const root = '../..';
+
 const
   should = require('should'),
   sinon = require('sinon'),
-  KuzzleError = require('../../src/KuzzleError'),
-  AbstractWrapper = require('../../src/protocols/abstract/common');
+  KuzzleError = require(`${root}/src/KuzzleError`),
```

Comment:
This kind of shortcut beak editor ability to automatically refactor imports when renaming files

Reply:
I wouldn't know, I don't use IDEs :expressionless: 

I'll revert that one, since this makes code marginally more readable.